### PR TITLE
PE-388 Add custom_report_format method

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
This branch adds the method custom_report format, that allows to return the answer for the given user and block.

### CODE REVIEW

File: activetable/activetable.py [hunk 1]
```diff
@@ -570,6 +577,58 @@ def generate_style_sheet(self, stylesheet):
...
+            total_width=sum(block._column_widths) if block._column_widths else None,
+            column_widths=block._column_widths,
+            head_height=block._row_heights[0] if block._row_heights else None,
+            thead=block.thead,
+            tbody=block.tbody,
+            score_type=block.score_type,
+            headers_style=re.sub("<[^<]*table>", "", block.headers_style) if block.custom_headers else None,
+        )
+        html = loader.render_template('templates/html/activetable_custom_report_format.html', context)
+        return html.replace('\n', "")
```
Comment (Squirrel18) [inline activetable/activetable.py]: Please don't mix single quotes and double quotes, use only one of them. @andrey-canon

File: activetable/templates/html/activetable_custom_report_format.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            <th class="active-table-custom-format-header" scope="col" style="border: 1px solid #c0c0c0; padding: 10px;">{{ cell }}</th>
+          {% endfor %}
+        </tr>
+      </thead>
+    {% endif %}
+    <tbody>
+      {% for row in tbody %}
+      <tr class="active-table-custom-format-row" style="height: {{ row.height }}px;">
+        {% for cell in row.cells %}
+          <td class="active-table-custom-format-cell"style="border: 1px solid #c0c0c0; padding: 10px;">{{cell.value}}</td>
```
Comment (Squirrel18) [inline activetable/templates/html/activetable_custom_report_format.html:21]: Maybe, all the inline css could be inside of the css file. @andrey-canon

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: activetable/templates/html/activetable_custom_report_format.html (hunk 2 above)
Comment (andrey-canon) [reply activetable/templates/html/activetable_custom_report_format.html:21]: the custom_report_format method only can return this html as string, doesn't include another file or fragment  and the string can't include style labels

Comment (Squirrel18) [reply activetable/templates/html/activetable_custom_report_format.html:21]: Ok.

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**